addons/mail/models/discuss/res_partner.py

```diff
@@ -106,5 +106,5 @@ def get_mention_suggestions_from_channel(self, channel_id, search, limit=8):
         store.add(channel, {"group_public_id": channel.group_public_id.id if channel.group_public_id else None})
         if allowed_group:
             for p in partners:
-                store.add(p, {"groups_id": [("ADD", (allowed_group & p.user_ids.groups_id).ids)]})
+                store.add(p, {"group_ids": [("ADD", (allowed_group & p.user_ids.group_ids).ids)]})
```

Comment:
`all_group_ids` (would need to rename the field in JS code too)

Reply:
Do you mean this or both?
```suggestion
                store.add(p, {"group_ids": [("ADD", (allowed_group & p.user_ids.all_group_ids).ids)]})
```

Reply:
Both of them provided the JS file is updated accordingly to the name on the left.